src/rpc/TKClient.h

```diff
@@ -521,4 +521,14 @@
  */
 - (void)getPairedDevices:(OnSuccessWithPairedDevices)onSuccess
                  onError:(OnError)onError;
+
+/**
+ * If more signatures is needed after endorsing a token, calls this method to notify
```

Comment:
Might be good to mention "We expect this to happen if user tried to endorse with a low-priv key on another device"